supervision/detection/line_counter.py

```diff
@@ -197,6 +202,50 @@ def __init__(
         self.text_padding: int = text_padding
         self.custom_in_text: str = custom_in_text
         self.custom_out_text: str = custom_out_text
+        self.display_in_count: bool = display_in_count
+        self.display_out_count: bool = display_out_count
+
+    def _annotate_count(
+        self,
+        frame: np.ndarray,
+        line_counter: LineZone,
+        text: str,
+        count_pos: bool,
+    ) -> None:
+        """This method is drawing the text on the frame.
+
+        Args:
+            frame (np.ndarray): The image on which the text will be drawn.
+            line_counter (LineCounter): The line counter
+                that will be used to draw the line.
+            text (str): The text that will be drawn.
+            count_pos (bool): Whether to display the in count or not.
+        """
+        text_width, text_height = cv2.getTextSize(
+            text, cv2.FONT_HERSHEY_SIMPLEX, self.text_scale, self.text_thickness
+        )[0]
+
+        center_text_anchor = Vector(
```

Comment:
I'd refactor the `_annotate_count`, and instead of passing the whole `line_counter: LineZone` object as an argument here, I'd pass the result of this code - `text_anchor: Point.`